app/components/AnalyzeComponent.js

```diff
@@ -58,13 +77,21 @@ export default class Analyze extends Component<Props, State> {
   handleChannelSelect: string => void;
   handleStepClick: (Object, Object) => void;
   handleDatasetChange: (Object, Object) => void;
+  handleBehaviorDatasetChange: (Object, Object) => void;
+  handleDependentVariableChange: (Object, Object) => void;
+  saveSelectedDatasets: () => void;
   handleStepClick: (Object, Object) => void;
 
   constructor(props: Props) {
     super(props);
     this.state = {
       activeStep: ANALYZE_STEPS.OVERVIEW,
       eegFilePaths: [{ key: '', text: '', value: '' }],
+      behaviorFilePaths: [{ key: '', text: '', value: '' }],
+      dependentVariables: [{ key: '', text: '', value: '' }],
```

Comment:
Does this conflict with the typing declared above? Shouldn't value be initialized as an object with a `name` and `dir` string?

Reply:
Corrected:)